src/main/java/org/jenkinsci/plugins/workflow/pipelinegraphanalysis/GenericStatus.java

```diff
@@ -46,6 +46,11 @@
     UNSTABLE,
 
     /**
+     * Not complete: still executing, in a node block, but the node block is in queue.
+     */
+    QUEUED,
```

Comment:
QUEUED != IN_PROGRESS ? 

Eh, I guess we do that for PAUSED_PENDING_INPUT, so w/e